admin-src/DiagnoseInfo/components/Response/styles.css

```diff
@@ -0,0 +1,34 @@
+.response {
+  width: 50%;
+  margin: 1em;
+  padding: 0.5em;
+  display: flex;
+  overflow: hidden;
+  justify-content: center;
+  border-radius: 0.5em;
+  -webkit-box-shadow: 0px 0px 6px 0px rgba(148,148,148,1);
+-moz-box-shadow: 0px 0px 6px 0px rgba(148,148,148,1);
+box-shadow: 0px 0px 6px 0px rgba(148,148,148,1);
```

Comment:
No need for prefixed properties. Change the extension of the file to `.scss`. It'll be a SASS stylesheet. SASS is _almost_ like CSS but gets compiled before Parcel adds it to the bundle. Sass handles these prefixes automatically.